cupy/core/_routines_math.pyx

```diff
@@ -23,6 +23,13 @@ if not cupy.cuda.runtime.is_hip:
 else:
     cub = None
 
+if cupy.cuda.cutensor_enabled:
+    import cupy_backends.cuda.libs.cutensor as cuda_cutensor
```

Comment:
Is this needed?

Reply:
yes, we want to move everything to `cupy_backends` for these libs.

Reply:
Ah OK.